tests/Block/Service/AbstractCategoriesBlockServiceTest.php

```diff
@@ -50,7 +50,7 @@ protected function setUp(): void
         $this->twig = $this->createMock(Environment::class);
         $this->contextManager = $this->createMock(ContextManagerInterface::class);
         $this->categoryManager = $this->createMock(CategoryManagerInterface::class);
-        $this->categoryAdmin = $this->createMock(CategoryAdmin::class);
+        $this->categoryAdmin = new CategoryAdmin('code', 'class', 'controller', $this->contextManager);
```

Comment:
```suggestion
        $this->categoryAdmin = new CategoryAdmin('code', \stdClass::class, 'controller', $this->contextManager);
```

phpstan will probably complain on a higher level as `class` is not a `class-string`

Reply:
I will and it too phpstan upgrade PR.